pylint/checkers/base/name_checker/checker.py

```diff
@@ -102,7 +85,7 @@ def _determine_function_name_type(
 
     :returns: One of ('function', 'method', 'attr')
     """
-    property_classes, property_names = _get_properties(config)
+    property_classes, property_names = utils.get_properties(config)
```

Comment:
This is not your PR, but it feels as if we should just use `property_classes` and always use inference.

Reply:
Are you going to address this issue?

Reply:
> This is not your PR

interpreted this as "not needed to do in this PR". I've received previous feedback that we should keep changes limited. Seeing how `get_properties` is used elsewhere it feels safer not to change htis at this time.

Reply:
Yeah let's make it easy to merge PR we don't need to fix all old issues in the affected code to merge only to fix one issue :)